### ISSUE TITLE
Are we missing a "data" folder here?

### ISSUE THREAD

**Issue by Leuvtern** [gboeing/urban-data-science#16]
Hi Professor Boeing,
Thank you very much for making this course available to public. When I try to go through the notebooks here, I found that they are referring to a "data" folder that contains some csv files, but I do not see such a folder under the PPD599 directory. I checked some of your other public courses such as PPD534 and PPD430 and saw that the csv files are usually provided in a "data" folder. Could you help add the data folder here, or point to another way of accessing the csv files? Thank you.

**Comment by yan-yuchen** [comment]
Yes, I think the dataset is indeed missing in the repository, causing the code to not run. Can the author provide data for learning?
